pysite/database.py

```diff
@@ -0,0 +1,44 @@
+# coding=utf-8
+__author__ = 'Joreth Cales'
+
+import os
+
+from flask import abort
+
+import rethinkdb
+
+
+class RethinkDB:
+
+    def __init__(self):
```

Comment:
Can we have a `loop_type` argument defaulted to `"gevent"`, or maybe a constant or something? We're gonna need to `rethinkdb.set_loop_type()` with it.